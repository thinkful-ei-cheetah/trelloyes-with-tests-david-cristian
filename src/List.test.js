import React from 'react';
import ReactDOM from 'react-dom';
import List from './List';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
    const section = document.createElement('section');

    ReactDOM.render(<List header="test" cards={["1", "2"]} />, section);

    ReactDOM.unmountComponentAtNode(section);
});

it('renders ui as expected', () => {
    const tree = renderer
    .create(<List header="test" cards={["1"]} />)
    .toJSON()

    expect(tree).toMatchSnapshot();
})