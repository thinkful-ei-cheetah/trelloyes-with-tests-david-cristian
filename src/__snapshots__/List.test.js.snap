// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ui as expected 1`] = `
<section
  className="List"
>
  <header
    className="List-header"
  >
    <h2>
      test
    </h2>
  </header>
  <div
    className="List-cards"
  >
    <div
      className="Card"
    >
      <h3 />
      <p />
      <button
        type="button"
      >
        Delete
      </button>
    </div>
    <button
      className="randomCard"
      type="button"
    >
      Add Random Card
    </button>
  </div>
</section>
`;
